### ISSUE TITLE
Can not reproduct the result in Table 2.

### ISSUE THREAD

**Issue by Zyriix** [NVlabs/edm#16]
I run the following training command on 8 v100.

torchrun --standalone --nproc_per_node=8 train.py --outdir=training-runs \
    --data=datasets/cifar10-32x32.zip --cond=0 --arch=ddpmpp

This gives me a FID at 2.08169 which is still far from FID in the paper(1.97).

I think this may caused by the random seeds(Random init in the code).
Is it possible to share the seed for reprodcuting the result in table2?

Any suggestiones?

**Comment by Newbeeer** [comment]
I observed the FID of EDM fluctuates largely. Did you report the lowest FID among checkpoints?

**Comment by Zyriix** [comment]
> 我观察到EDM的FID波动很大。您是否报告了检查站中 FID 最低的？

I found this in some paper that try to retrain edm. I got the lowest FID(2.04053) at around 16w ckpt. 
I think the fluctuation is caused by the lower ema rate

> I observed the FID of EDM fluctuates largely. Did you report the lowest FID among checkpoints?

**Comment by Zyriix** [comment]
> I observed the FID of EDM fluctuates largely. Did you report the lowest FID among checkpoints?

I found this in some papers that try to retrain Edm. I got the lowest FID(2.04053) at around 16w ckpt.
I think the fluctuation is caused by lower ema rate and larger learning rate.

**Comment by yuanzhi-zhu** [comment]
@Zyriix To calculate FID we have to first generate, saying 50000 samples. The random seed can influence the generation process of these samples hence the FID value I guess

**Comment by Zyriix** [comment]
> @Zyriix To calculate FID we have to first generate, saying 50000 samples. The random seed can influence the generation process of these samples hence the FID value I guess

Most works use the same sample seeds just like this repo:
0~49999, 50000~99999, 100000~149999
and report the lowest one.

**Comment by Liyuan-Liu** [comment]
--arch=ncsnpp performs much better than VP architecture

**Comment by Jiuyi-Mines** [comment]
Hey guys,

Did anyone solve this problem? I cannot retrain EDM with "--cond=0 --arch=ddpmpp" on CIFAR10 to get the reported FID (1.97), either.
